src/seedu/addressbook/commands/GoodbyeCommand.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.commands;
+
+
+/**
+ * print goodbye message.
+ */
+public class GoodbyeCommand extends Command {
+
+    public static final String COMMAND_WORD = "goodbye";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": prints goodbye message.\n"
+            + "Example: " + COMMAND_WORD;
+
+    @Override
```

Comment:
Missing header comment. All non-trivial methods (in this case your enhancement functionality) should have java doc format header comments.